.gitignore

```diff
@@ -1,5 +1,17 @@
 node_modules/
 build/
+public/
 coverage/
+
 .vscode/
+.cache/
+
+# Log files
 lerna-debug.log
+
+# Environment Variables
+.env
+.env.production
+
+# gatsby-source-nacelle
+examples/*/nacelle-fragments/*.graphql
```

Comment:
Feel free to just add this within the local `.gitignore` in the package itself rather than try to account for everything in the root of the project

Reply:
Will do!